flask_project/campaign_manager/api.py

```diff
@@ -126,6 +128,41 @@ def get(self):
             'participant_total': len(participants)
         }
 
+
+class CampaignContributors(Resource):
+    """Show Campaign Contributors."""
+
+    def get_campaign(self, uuid):
+        """Return campaign."""
+        return Campaign(uuid=uuid)
+
+    def get(self, uuid):
+        """Get total contributors."""
+
+        campaign = self.get_campaign(uuid)
+        mapper_buildings = MapperEngagement(
```

Comment:
instead on duplicated code like this
you could use loop in here
and also, you could use feature in here

https://github.com/hotosm/field-campaigner/blob/master/reporter/queries.py#L3

```
user = []
for feature in features:
mapper = MapperEngagement(campaign=campaign, feature='roads')
mapper.run()
data = mapper_roads.get_function_data()
for entry in range(len(data1)):
     if data1[entry]['name'] not in user:
            user.append(data1[entry]['name'])




```